modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -94,6 +94,11 @@ def find_poa_by_participant_id(id)
                    key: 'return')
     end
 
+    def find_poas
+      make_request(endpoint: 'StandardDataWebServiceBean/StandardDataWebService', action: 'findPOAs', body: nil,
+                   key: 'PowerOfAttorneyDTO')
```

Comment:
cf. https://github.com/department-of-veterans-affairs/bgs-catalog/blob/main/StandardDataWebServiceBean/StandardDataWebService/StandardDataWebServicePortBinding/findPOAs/request.xml

Reply:
can we move this to a StandardDataWebService? similar to the BenefitClaimService or ClaimManagementService?

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/18195/commits/f49406d0fff8d06ac961ef7ae6c59b2c5d3e0047